<launch>
  <arg name="veh" default="blueduckie" />
  <group ns="$(arg veh)">
    <node pkg="temp_camera" type="temp_camera_node.py" name="camera_node" output="screen" respawn="true">
      <param name="veh" value="$(arg veh)" />
    </node>
  </group>
</launch>
